### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by MarvinCrepin**

### CODE REVIEW

File: templates/_navbar.html.twig [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<body class="align">
```
Comment (rocambille) [inline templates/_navbar.html.twig]: 1 body per page is enough: you already have 1 in base

File: templates/base.html.twig [hunk 2]
```diff
@@ -14,6 +14,7 @@
 			{{ encore_entry_script_tags('app') }}
 		{% endblock %}
 	</head>
+	{% include '_navbar.html.twig' %}
```
Comment (rocambille) [inline templates/base.html.twig]: content should be in body

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
conflicts to resolve and errors to fix : https://github.com/WildCodeSchool/reims-202109-php-project3-marche-conclu/runs/4468760169?check_suite_focus=true

**Comment by MarvinCrepin** [comment]
> conflicts to resolve and errors to fix : https://github.com/WildCodeSchool/reims-202109-php-project3-marche-conclu/runs/4468760169?check_suite_focus=true

i dont know how to fix the "react" error its a weird issue

**Comment by rocambille** [comment]
if you put the error message in googl (not the warning about react stuff, the TypeError: Cannot read properties of null (reading 'first')): https://www.google.com/search?channel=fs&client=ubuntu&q=sass-lint+Cannot+read+properties+of+null+%28reading+%27first%27%29

the first result gives some light: https://github.com/sasstools/sass-lint/issues/1161

File: assets/styles/app.scss [hunk 3]
```diff
@@ -1,22 +1,86 @@
...
...9uLWJvcmRlci1yYWRpdXM6IDAuMjVlbTtcblxuICAtLW5hdmlnYXRpb24tYW5jaG9yLXBhZGRpbmc6IDEuNWVtO1xufVxuXG4ubmF2aWdhdGlvbiB7XG4gICAgYmFja2dyb3VuZC1jb2xvcjogI2YwZjZmZjtcbiAgYmFja2dyb3VuZC1jb2xvcjogdmFyKC0tbmF2aWdhdGlvbi1iYWNrZ3JvdW5kLWNvbG9yKTtcbiAgYm9yZGVyLXJhZGl1czogdmFyKC0tbmF2aWdhdGlvbi1ib3JkZXItcmFkaXVzKTtcbiAgYm94LXNoYWRvdzogMCAwLjVlbSAxZW0gcmdiYSgwLCAwLCAwLCAwLjMpO1xuICB3aWR0aDogMTAwJTtcbiAgYm90dG9tOiAwO1xuICBwb3NpdGlvbjogZml4ZWQ7XG59XG5cbi5uYXZpZ2F0aW9uIHVsIHtcbiAgbGlzdC1zdHlsZTogbm9uZTtcbiAgbWFyZ2luOiAwO1xuICBwYWRkaW5nOiAwO1xufVxuXG4ubmF2aWdhdGlvbiBhIHtcbnBhZGRpbmc6IDEuNXJlbTtcbiAgZGlzcGxheTogYmxvY2s7XG4gIHBhZGRpbmc6IHZhcigtLW5hdmlnYXRpb24tYW5jaG9yLXBhZGRpbmcpO1xuICBwb3NpdGlvbjogcmVsYXRpdmU7XG4gIHRleHQtZGVjb3JhdGlvbjogbm9uZTtcbn1cblxuLm5hdmlnYXRpb24gYTpmb2N1czo6YWZ0ZXIsXG4ubmF2aWdhdGlvbiBhOmhvdmVyOjphZnRlciB7XG4gIG9wYWNpdHk6IDE7XG59XG5cbi5uYXZpZ2F0aW9uIGE6OmFmdGVyIHtcbiAgYmxvY2stc2l6ZTogMTAwJTtcbiAgYm94LXNoYWRvdzogMCAwLjVlbSAxZW0gcmdiYSgwLCAwLCAwLCAwLjMpO1xuICBpbmxpbmUtc2l6ZTogMTAwJTtcbiAgb3BhY2l0eTogMDtcbiAgdHJhbnNpdGlvbjogb3BhY2l0eSAwLjNzO1xufVxuXG4ubmF2aWdhdGlvbi0taW5saW5lIHVsIHtcbiAgZGlzcGxheTogZmxleDtcbn1cbi8qIEVORCBPRiBOQVZCQVIgKi9cbiJdLCJuYW1lcyI6W10sInNvdXJjZVJvb3QiOiIifQ==*/
```
Comment (rocambille) [inline assets/styles/app.scss]: this shouldn't be in the scss file. it's good to have removed the custom properties, but you may have gone with sass variables ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

**Pull request merged**